src/main/java/org/springframework/data/couchbase/repository/query/CouchbaseQueryMethod.java

```diff
@@ -173,8 +208,28 @@ public boolean isCountQuery() {
 		return getName().toLowerCase().startsWith("count");
 	}
 
+	public boolean isPageableQuery() {
+		for (Parameter p : getParameters()) {
+			if (org.springframework.util.ClassUtils.isAssignable(PageRequest.class, p.getType()))
+				;
```

Comment:
This doesn't look right -- it will return true if the there is at least one parameter of any time. Missing curly braces?